### PR TITLE
db/fixed-dark-mode-demo

### PR DESCRIPTION
**Pull request by sebastianbochan**

### CODE REVIEW

File: samples/dashboards/demos/component-highcharts/demo.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
+// Necessary to enable styled mode in order to properly style the
+// chart depending on the theme.
+Highcharts.setOptions({
```
Comment (karolkolodziej) [inline samples/dashboards/demos/component-highcharts/demo.js:3]: Why don't we enable that for each HighchartsComponent?

File: samples/dashboards/demos/axes-sync/demo.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
-@import "https://code.highcharts.com/css/highcharts.css";
-@import "https://code.highcharts.com/css/dashboards/gui.css";
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/css/dashboards/gui.css");
```
Comment (karolkolodziej) [inline samples/dashboards/demos/axes-sync/demo.css:2]: It isn't fully solved.
image

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

**Review by karolkolodziej — APPROVED** [review]
The code looks and works, but please check the conflicts and failing tests in Cypress before merging.

**Review by bre1470 — CHANGES_REQUESTED** [review]
`dashboards/cypress/add-layout` needs an update as well.

Screenshot 2023-05-22 at 13 10 37

**Review by bre1470 — APPROVED** [review]
@sebastianbochan
Okay, I see.

I also find it strange that there is a line in the menu itself. It looks like the demo is not up to date with styling in general.

**Pull request merged**